substratevm/src/com.oracle.svm.hosted/src/com/oracle/svm/hosted/image/NativeImageDebugInfoProvider.java

```diff
@@ -195,6 +196,17 @@ protected static ResolvedJavaType getJavaType(HostedType hostedType, boolean wan
         return hostedType.getWrapped().getWrapped();
     }
 
+    protected static ResolvedJavaType getJavaType(AnalysisType analysisType, boolean wantOriginal) {
```

Comment:
Why do you need this? Where are you seeing Analysis types pop up? I think you ought only to be seeing HostedTypes.

Reply:
Hmmm, you are right. I guess I somehow thought I should include it like we do in `computeFullFilePath` (but there it's just a step after unwrapping a `HostedType`). Anyhow this is now removed.